std/commerce/vtex/transform.ts

```diff
@@ -66,22 +66,26 @@ export const toProductPage = (
   };
 };
 
-export const inStock = (offer: CommertialOffer) => offer.AvailableQuantity > 0;
+export const inStock = (offer: Offer) =>
+  offer.availability === "https://schema.org/InStock";
 
 // Smallest Available Spot Price First
-export const bestOfferFirst = (
-  a: SellerVTEX,
-  b: SellerVTEX,
-) => {
-  if (inStock(a.commertialOffer) && !inStock(b.commertialOffer)) {
+export const bestOfferFirst = (a: Offer, b: Offer) => {
+  if (inStock(a) && !inStock(b)) {
     return -1;
   }
 
-  if (!inStock(a.commertialOffer) && inStock(b.commertialOffer)) {
+  if (!inStock(a) && inStock(b)) {
     return 1;
   }
 
-  return a.commertialOffer.spotPrice - b.commertialOffer.spotPrice;
+  return a.price - b.price;
+};
+
+const getHighPriceIndex = (offers: Offer[]) => {
+  let it = offers.length - 1;
+  for (; it >= 0 && !inStock(offers[it]); it--);
```

Comment:
```suggestion
  for (let it = offers.length - 1; it >= 0 && !inStock(offers[it]); it--);
```

Reply:
eu preciso retornar o `it` ali na linha 88, dai preciso declarar ele fora do `for`